cocotb_build_libs.py

```diff
@@ -47,12 +47,10 @@ def build_extension(self, ext):
                 "install_name_tool",
                 "-add_rpath",
                 "@loader_path",
-                os.path.join(self.build_lib, ext._file_name),
-            ], check=True)
-            subprocess.run([
-                "install_name_tool",
                 "-add_rpath",
                 "@loader_path/libs",
+                "-add_rpath",
+                sysconfig.get_config_var("LIBDIR"),
```

Comment:
I assume you combined these into a single invocation because it was tidier, not because it was necessary?

Would be great if this commit message could mention that.

Reply:
Nevermind, I see this gets removed in a later commit anyway - guess we should squash this PR when we merge.